docker/part-2/6/6.txt

```diff
@@ -0,0 +1,7 @@
+1. the layering mechanism helps make the development faster by caching each layer in the building process, reusing layer components,
+once an image is built it is consistent and layers can br built in parallel.  
+
+2. method 1 is better to run command one after the other because method 2 makes the execution in parallel in contrast to method 1 that 
```

Comment:
docker runs the layers in parallel only when possible- not always. the big difference in this question is the amount of memory space each layer is adding